.travis.yml

```diff
@@ -19,5 +22,14 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 
+# For a PR TRAVIS_BRANCH is set to the target branch which is master.
+# We need to make sure it's not a PR when releasing.
 script:
-   - sbt ++$TRAVIS_SCALA_VERSION ciBuild
+  - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ];
+    then
+      docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+      sbt ++$TRAVIS_SCALA_VERSION ciRelease;
```

Comment:
Ah thanks to pointing me to that. I wasn't sure how ciRelease was used and wrongly assumed it hadn't been hooked in. I'm going to revert this back - the docker publish will happen as part of ciRelease so we can keep the same process as we have now, in the future at least we know it's easy to get working with Travis 😸